store/auth.go

```diff
@@ -158,3 +160,40 @@ func ReadStoreToken() (*StoreToken, error) {
 
 	return &readStoreToken, nil
 }
+
+// RequestPackageAccessMacaroon requests a macaroon for accessing package data from the ubuntu store
+func RequestPackageAccessMacaroon() (string, error) {
+	emptyJsonData := "{}"
+	req, err := http.NewRequest("POST", myappsPackageAccessAPI, strings.NewReader(emptyJsonData))
+	req.Header.Set("accept", "application/json")
+	req.Header.Set("content-type", "application/json")
+	if err != nil {
+		return "", err
+	}
+
+	client := &http.Client{}
+	resp, err := client.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	// check return code, error on anything !200
+	if !httpStatusCodeSuccess(resp.StatusCode) {
+		// unexpected result, bail
+		return "", fmt.Errorf("failed to get store macaroon: %v (%v)", resp.StatusCode, resp)
```

Comment:
That will be a number plus a string representation of a Go value, which isn't going to be very clear to the user.

Won't the store return something slightly more meaningful in general for issues it reports?